plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/FileManagementUtil.java

```diff
@@ -0,0 +1,399 @@
+        InputStream is = null;
+        FileOutputStream fos = null;
+
+        try {
+            is = new FileInputStream(src);
+            fos = new FileOutputStream(dest);
+            int c = 0;
+            byte[] array = new byte[1024];
+            while ((c = is.read(array)) >= 0) {
+                fos.write(array, 0, c);
+            }
+        } catch (FileNotFoundException e) {
+            logger.log(Level.FINE, HumantaskEditorConstants.ERROR_COPYING_FILES, e);
+        } catch (IOException e) {
+            logger.log(Level.FINE, HumantaskEditorConstants.ERROR_COPYING_FILES, e);
+        } finally {
+            try {
+                fos.close();
+                is.close();
+            } catch (IOException e) {
+                logger.log(Level.FINE, HumantaskEditorConstants.ERROR_COPYING_FILES, e);
+            }
+        }
+    }
+
+    public static File createFileAndParentDirectories(String fileName) throws IOException {
+        File file = new File(fileName);
+        File parent = file.getParentFile();
+        if (!parent.exists()) {
+            parent.mkdirs();
+        }
+        file.createNewFile();
+        return file;
+    }
+
+    public static boolean deleteDir(File dir) {
+        if (dir.isDirectory()) {
+            String[] children = dir.list();
+            for (int i = 0; i < children.length; i++) {
+                boolean success = deleteDir(new File(dir, children[i]));
+                if (!success) {
+                    return false;
+                }
+            }
+        }
+        return dir.delete();
+    }
+
+    public static void deleteDirectories(File dir) {
+        File[] children = dir.listFiles();
+        for (int i = 0; i < children.length; i++) {
+            if (children[i].list() != null && children[i].list().length > 0) {
+                deleteDirectories(children[i]);
+            } else {
+                children[i].delete();
+            }
+        }
+        dir.delete();
+    }
+
+    public static void deleteDirectories(String dir) {
+        File directory = new File(dir);
+        deleteDirectories(directory);
+    }
+
+    public static void createTargetFile(String sourceFileName, String targetFileName) throws IOException {
+        createTargetFile(sourceFileName, targetFileName, false);
+    }
+
+    public static void createTargetFile(String sourceFileName, String targetFileName, boolean overwrite)
+            throws IOException {
+        File idealResultFile = new File(targetFileName);
+        if (overwrite || !idealResultFile.exists()) {
+            FileManagementUtil.createFileAndParentDirectories(targetFileName);
+            FileManagementUtil.copyFile(sourceFileName, targetFileName);
+        }
+    }
+
+    public static boolean createDirectory(String directory) {
+        // Create a directory; all ancestor directories must exist
+        boolean success = (new File(directory)).mkdir();
+        if (!success) {
```

Comment:
This if block does nothing here. At least put a log entry here, or remove the if.